src/components/Footer.jsx

```diff
@@ -3,13 +3,13 @@ import styled from 'styled-components'
 
 function Footer() {
   return (
-    <div className='Navbar'>
     <NavContainer>
+    <>
```

Comment:
You dont need this fraction element, because u use allready your NavContainer

Reply:
tanks i will delete the unused fraction elements